src/map/GMap.js

```diff
@@ -9,17 +9,33 @@
     function GMap(target) {
         Graph.call(this);
         IGMap.call(this);
+
+        this.layout("none");
+
+        this._markers = [];
     }
     GMap.prototype = Object.create(Graph.prototype);
     GMap.prototype._class += " map_GMap";
     GMap.prototype.implements(IGMap.prototype);
 
+    GMap.prototype.data = function (_) {
+        if (arguments.length) {
+            this.graphData.nodeValues().forEach(function (item) {
+                if (item._marker) {
+                    item._marker.setMap(null);
+                }
+            });
+        }
+        var retVal = Graph.prototype.data.apply(this, arguments);
+        return retVal;
+    };
+
     GMap.prototype.enter = function (domNode, element, d) {
         Graph.prototype.enter.apply(this, arguments);
 
         this._googleMap = new google.maps.Map(d3.select(this._target).node(), {
-            zoom: 3,
-            center: new google.maps.LatLng(41.850033, -87.6500523),
+            zoom: 4,
+            center: new google.maps.LatLng(42.877742, -97.380979),
```

Comment:
Possibly, but need somewhere to default it to and this one is practical at least.